docs/docs/about/why-cli.mdx

```diff
@@ -2,8 +2,47 @@
 sidebar_position: 1
 ---
 
-# Why this CLI
+# Why this CLI?
 
 CLI for Microsoft 365 is a cross-platform CLI that allows users on any platform to manage various configuration settings of Microsoft 365.
 
-While building solutions for Microsoft 365 expands beyond the Windows operating system, managing many of the platform settings is possible only through PowerShell on Windows. As more and more users work on non-Windows machines, it's inconvenient for them to have to use a Windows virtual machine to configure their tenants. The CLI for Microsoft 365 allows them to configure their tenants no matter which operating system they use.
+- CLI for Microsoft 365 offers a variety of commands from many different areas of Microsoft 365 like: [SharePoint Online](../cmd/spo/spo-get), [Teams](../cmd/teams/app/app-install), [Viva engage](../cmd/viva/engage/engage-search), [Entra](../cmd/entra/administrativeunit/administrativeunit-add), [Power Apps](../cmd/pa/app/app-export), [Power Automate](../cmd/flow/flow-disable), [Power Platform](../cmd/pp/aibuildermodel/aibuildermodel-get), [OneDrive](../cmd/onedrive/onedrive-list), [Planner](../cmd/planner/bucket/bucket-add), [OneNote](../cmd/onenote/notebook/notebook-add), [Outlook](../cmd/outlook/mail/mail-send), [SharePoint Embedded](../cmd/spe/containertype/containertype-add) and many more.
+- Intuitive! If you would like to `add` a new `SharePoint Online` `list` simply write `m365 spo list add` and if you need to do a different action like modifying the list or retrieving it simply change the verb to: `get`, `set`, `remove` etc. 
+- Interactive by default. Starting to use a new CLI is never easy, that's why CLI for Microsoft 365 offers an interactive mode that actively asks (prompts) you for the required options of a command you want to run. You may also [configure autocomplete for commands](../user-guide/completion.mdx) and options that you're typing in the prompt for almost any terminal.
+- [Different `output` modes](../user-guide/cli-output-mode.mdx) allow you to choose how you want to see the results of your commands. You can choose between `json` which is perfect to be used in a script when the output of one command will be used as an input to another command, `text` which is perfect for human reading, and `csv` or even `md` (markdown) which is perfect to create a report based outcome.
```

Comment:
Let's avoid repeating _perfect_ each time. We can use other words like _convenient_, _helpful_, _optimized_, _suitable_, etc.